### DISCUSSION TITLE
Fehler bei Software update

### DISCUSSION THREAD

**Discussion by buerstet4** [tbnobody/OpenDTU#1533] [Q&A]
Ich wollte ein Software Update von 23.10.9 auf 23.11.16 machen, habe es mehrfach versucht, auch mehrmals die neue Software runter geladen, aber leider kommt immer dieser Fehler, den ich nicht zuordnen kann
![2023-11-28 17_33_48-OpenDTU – Opera](https://github.com/tbnobody/OpenDTU/assets/125080572/2ea06ffb-4ac2-47eb-9f26-245e0277c67d)

**Comment by Vaclar01** [comment]
Folgendes könnte helfen. 

https://github.com/tbnobody/OpenDTU/blob/master/docs/UpgradePartition.md


Hier ein anderer Bugreport indem das erläutert wird.
https://github.com/tbnobody/OpenDTU/issues/1501

**Comment by buerstet4** [comment]
Danke für die Info, allerdings hat bei mir nur kpl. neu flashen geholfen.
